src/psyclone/dynamo0p3.py

```diff
@@ -3134,6 +3124,9 @@ def _invoke_declarations(self, parent):
         '''
         Generate the necessary PSy-layer declarations for all column-wise
         operators and their associated parameters.
+        Note: PSy layer in LFRic does not modify the CMA operator objects.
+        Hence, their Fortran intents are always "in" (the data pointed to
```

Comment:
Please update this docstring along the lines of the one for LMA quantities.

Reply:
Done.